meilisearch/tests/search/errors.rs

```diff
@@ -321,6 +321,25 @@ async fn search_bad_facets() {
     // Can't make the `attributes_to_highlight` fail with a get search since it'll accept anything as an array of strings.
 }
 
+#[actix_rt::test]
+async fn search_non_filterable_facets() {
+    let server = Server::new().await;
+    let index = server.index("test");
+    index.update_settings(json!({"filterableAttributes": ["title"]})).await;
+
+    let (response, code) = index.search_post(json!({"facets": ["doggo"]})).await;
+    snapshot!(code, @"400 Bad Request");
+    snapshot!(json_string!(response), @r###"
+    {
+      "message": "Invalid facet distribution, the fields `doggo` are not set as filterable.",
+      "code": "invalid_search_facets",
+      "type": "invalid_request",
+      "link": "https://docs.meilisearch.com/errors#invalid-search-facets"
+    }
+    "###);
+    // Can't make the `attributes_to_highlight` fail with a get search since it'll accept anything as an array of strings.
```

Comment:
Could you write a test with a `search_get` since you actually can in this case

Reply:
done